buildkitd/buildkitd.go

```diff
@@ -84,7 +84,11 @@ func NewClient(ctx context.Context, console conslogging.ConsoleLogger, image, co
 		remoteConsole := console
 		if settings.SatelliteName != "" {
 			remoteConsole = console.WithPrefix("satellite")
-			remoteConsole.Printf("Connecting to %s...", settings.SatelliteDisplayName)
+			selfHostedInfo := ""
+			if settings.SatelliteIsSelfHosted {
+				selfHostedInfo = fmt.Sprintf(" (self-hosted at %s)", settings.BuildkitAddress)
+			}
+			remoteConsole.Printf("Connecting to %s...%s", settings.SatelliteDisplayName, selfHostedInfo)
```

Comment:
this line includes `selfHostedInfo`; however doesn't fall under the `if settings.SatelliteIsSelfHosted {` block; is this intentional?

i.e. could this line be printed in non-self-hosted cases too?